app/src/main/java/com/worldsnas/starplayer/di/modules/MusicProviderModule.kt

```diff
@@ -0,0 +1,29 @@
+package com.worldsnas.starplayer.di.modules
+
+import android.app.Application
+import com.worldsnas.starplayer.api.WebServiceApi
+import com.worldsnas.starplayer.model.LocalMusicProvider
+import com.worldsnas.starplayer.model.LocalMusicProviderImpl
+import com.worldsnas.starplayer.model.MusicRepository
+import com.worldsnas.starplayer.model.MusicRepositoryImpl
+import dagger.Module
+import dagger.Provides
+
+@Module
+object MusicProviderModule {
```

Comment:
vaghti @Inject mizarim balaye constrcutor dige niazi be in file nist

Reply:
vaghti pak mikonm khata mide ke provide nashodan